Foundation/NSAttributedString.swift

```diff
@@ -7,9 +7,16 @@
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 
+import CoreFoundation
 
 public class NSAttributedString : NSObject, NSCopying, NSMutableCopying, NSSecureCoding {
     
+    private var _cfAttributedString: CFAttributedString!
```

Comment:
this would make NSAttributedString to be a has-a versus an is-a relationship to CFAttributedString; which is a variation (however it probably does not matter since there are not any CF apis that could take a attributed string.

It would be better to have this not be null resettable; removing the `!` would be preferable.